src/components/core/DpUpload/utils/GetFileIdsByHash.js

```diff
@@ -7,9 +7,9 @@ import { dpApi } from '../../../../lib'
  *
  * @returns {Array<Ids>}
  */
-function getFileIdsByHash (hashes) {
+function getFileIdsByHash (hashes, route) {
   return dpApi.get(
-    Routing.generate('api_resource_list', { resourceType: 'File' }),
+    route,
```

Comment:
we should rework this one separately. This logic isn't generic enough. Maybe this even doesn't belong here bot in the core